### PR TITLE
Fixes typos in comments and docs

### PR DESCRIPTION
**Pull request by mpgussert**
## Checklist

- [X] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [X] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

**Comment by Toni-SM** [comment]
@mpgussert 

Takes advantage of this PR to correct the 9 references to `articultion` in source code and docs :sweat_smile:

**Comment by mpgussert** [comment]
fixed!! 

Sorry!  I need to get into the habit of checking my personal email for work stuff!  I see a lot of comment replies too.... oops...

**Review by Dhoeller19 — APPROVED** [review]

File: docs/source/migration/migrating_from_isaacgymenvs.rst [hunk 1]
```diff
@@ -660,7 +660,7 @@ the need to set simulation parameters for actors in the task implementation.
 |     self._create_ground_plane()                                        |         copy_from_source=False)                                     |
 |     self._create_envs(self.num_envs,                                   |     self.scene.filter_collisions(                                   |
 |         self.cfg["env"]['envSpacing'],                                 |         global_prim_paths=[])                                       |
-|         int(np.sqrt(self.num_envs)))                                   |     # add articultion to scene                                      |
+|         int(np.sqrt(self.num_envs)))                                   |     # add articulation to scene                                      |
```
Comment (Dhoeller19) [inline docs/source/migration/migrating_from_isaacgymenvs.rst]:
```suggestion
|         int(np.sqrt(self.num_envs)))                                   |     # add articulation to scene                                     |
```

File: docs/source/migration/migrating_from_omniisaacgymenvs.rst [hunk 2]
```diff
@@ -219,7 +219,7 @@ will automatically be created for the actor. This avoids the need to separately
 |     self._cartpoles = ArticulationView(                                      |     # clone, filter, and replicate                                     |
 |                  prim_paths_expr="/World/envs/.*/Cartpole",                  |     self.scene.clone_environments(copy_from_source=False)              |
 |                  name="cartpole_view", reset_xform_properties=False          |     self.scene.filter_collisions(global_prim_paths=[])                 |
-|     )                                                                        |     # add articultion to scene                                         |
+|     )                                                                        |     # add articulation to scene                                         |
```
Comment (Dhoeller19) [inline docs/source/migration/migrating_from_omniisaacgymenvs.rst]:
```suggestion
|     )                                                                        |     # add articulation to scene                                        |
```

**Pull request merged**